### PR TITLE
updated rel 4.0.0 installation steps

### PR DESCRIPTION
**Pull request by G33tha**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @G33tha,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://849.qa.docs.sunbird.org/4.0.0/ .<br/> **Note:** Only _4.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://849.qa.docs.sunbird.org/4.0.0/developer-docs/server-installation/ansible-variables/ <br /> - http://849.qa.docs.sunbird.org/4.0.0/developer-docs/server-installation/build-and-deploy/ <br /> - http://849.qa.docs.sunbird.org/4.0.0/developer-docs/server-installation/jenkins-setup/ <br /> - http://849.qa.docs.sunbird.org/4.0.0/developer-docs/server-installation/prerequisites/ <br />  <br/> **Modified sections are -** <br />

**Comment by G33tha** [comment]
@Sunitashelar please merge

File: developer-docs/server-installation/prerequisites.md [hunk 1]
```diff
@@ -34,6 +34,25 @@ Before you start the installation process, ensure that you have the required inf
 | Other Services | 4core 16G 60G HDD | 1 |
 | Basic Load Balancers | - | 2 (Optional) |
 
+
+##### Automation on azure infrastructure creation
```
Comment (NupurSancheti) [inline developer-docs/server-installation/prerequisites.md:38]: Change to Heading 4: Automation on Azure Infrastructure Creation

Comment (Sunitashelar) [reply developer-docs/server-installation/prerequisites.md]: Done

**Pull request merged**